MMM-voice.js

```diff
@@ -220,10 +222,51 @@ Module.register('MMM-voice', {
             MM.getModules().enumerate((module) => {
                 module.hide(1000);
             });
+            // tell other modules all hidden
+            this.sendNotification('NOW_ASLEEP','[]')
         } else if (notification === 'SHOW') {
             MM.getModules().enumerate((module) => {
                 module.show(1000);
             });
+            // tell other modules all shown
+            this.sendNotification('NOW_AWAKE')
+				} else if (notification === 'SLEEP_HIDE') {
+						// sleep by hiding (energyStar monitors)
+            let self=this;
+            let list=[];
```

Comment:
What is the difference between `list` and `previouslyHidden`? For me there is no need to create a duplicated list, use only the module variable previouslyHidden

Reply:
removed list